packages/sources/dlc-btc-por/README.md

```diff
@@ -1,22 +1,23 @@
-| :-------: | :--------------------: | :---------------------------------------------------------------------------------------: | :----: | :-----------------------------: | :-------: |
-|    ✅     |        RPC_URL         |                          The RPC URL to connect to the EVM chain                          | string |                                 |           |
-|    ✅     |        CHAIN_ID        |                              The EVM chain id to connect to                               | number |                                 |           |
-|    ✅     |      DLC_CONTRACT      |                        Contract address to fetch all funded vaults                        | string |                                 |           |
-|    ✅     |   EVM_RPC_BATCH_SIZE   |                  Number of vaults to fetch from a DLC contract at a time                  | number |                                 |   `100`   |
-|    ✅     |    BITCOIN_RPC_URL     |                                THE RPC URL of bitcoin node                                | string |                                 |           |
-|    ✅     |    BITCOIN_NETWORK     |                                   Bitcoin network name                                    |  enum  | `mainnet`, `regtest`, `testnet` | `mainnet` |
-|    ✅     |     CONFIRMATIONS      |                      The number of confirmations to query data from                       | number |                                 |    `6`    |
-|    ✅     | BITCOIN_RPC_GROUP_SIZE |         The number of concurrent RPC calls to BITCOIN_RPC_URL to make at a time.          | number |                                 |   `30`    |
-|           | BACKGROUND_EXECUTE_MS  | The amount of time the background execute should sleep before performing the next request | number |                                 |  `10000`  |
+| Required? |          Name           |                                             Description                                              |  Type  |             Options             |  Default  |
+| :-------: | :---------------------: | :--------------------------------------------------------------------------------------------------: | :----: | :-----------------------------: | :-------: |
+|           |         RPC_URL         |     [DEPRECATED]. Please use {NETWORK}\_RPC_URL instead. The RPC URL to connect to the EVM chain     | string |                                 |           |
+|           |        CHAIN_ID         |         [DEPRECATED]. Please use {NETWORK}\_CHAIN_ID instead.The EVM chain id to connect to          | number |                                 |           |
+|           |      DLC_CONTRACT       | [DEPRECATED]. Please use {NETWORK}\_DLC_CONTRACT instead.Contract address to fetch all funded vaults | string |                                 |           |
```

Comment:
here too

Reply:
Why are we changing readme in PR? We have a GH that update these automatically after merge

Reply:
this EA was added to the readme generator blacklist: packages/scripts/src/generate-readme/readmeBlacklist.json

these env var changes are similar to your additions for harris-and-trotter in https://github.com/smartcontractkit/external-adapters-js/commit/5e53f62b47c9c523b30bad8e0edc402b2adad8b9